package.json

```diff
@@ -38,7 +38,7 @@
     "@babel/preset-typescript": "^7.12.7",
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
-    "@ethersphere/bee-js": "^0.5.0",
+    "@ethersphere/bee-js": "^0.8.1",
```

Comment:
`0.9.0` is the latest, is it not what you need to use?

Reply:
nope, the 0.6.0 Bee version compatibility will be introduced in PR that will resolve issue #35